OcarinaSongs.py

```diff
@@ -265,77 +322,60 @@ def get_random_song():
 
 
 # create a list of 12 songs, none of which are sub-strings of any other song
-def generate_song_list():
-    songs = []
-
-    for _ in range(12):
-        while True:
+def generate_song_list(world):
+    fixed_songs = {name: Song.from_str(notes) for name, notes in world.distribution.configure_songs().items()} #TODO length checks?
```

Comment:
Did the length checks get added?

Reply:
They did (lines 328–331), I just forgot to remove the todo comment.